samples/nrf9160/azure_iot_hub/prj.conf

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# General config
+CONFIG_LOG=y
+CONFIG_LOG_IMMEDIATE=y
+
+# NEWLIB C
+CONFIG_NEWLIB_LIBC=y
+
+# Network
+CONFIG_NETWORKING=y
+CONFIG_NET_NATIVE=n
+
+# LTE link control
+CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LTE_NETWORK_MODE_LTE_M=y
+CONFIG_LTE_AUTO_INIT_AND_CONNECT=n
+
+# BSD library
+CONFIG_BSD_LIBRARY=y
+
+# AT Host
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_AT_HOST_LIBRARY=y
+CONFIG_AT_CMD_THREAD_STACK_SIZE=2048
+
+# Azure IoT Hub library
+CONFIG_AZURE_IOT_HUB=y
+CONFIG_AZURE_IOT_HUB_HOSTNAME=""
+CONFIG_AZURE_IOT_HUB_DEVICE_ID=""
+CONFIG_AZURE_IOT_HUB_SEC_TAG=
+
+# Uncomment and configure the options below to use DPS for device provisioning
+# CONFIG_AZURE_IOT_HUB_DPS=y
+# CONFIG_AZURE_IOT_HUB_DPS_ID_SCOPE=""
+
+# Uncomment to get more verbose logging when debugging
+# CONFIG_AZURE_IOT_HUB_LOG_LEVEL_DBG=y
+
+# LED control
+CONFIG_DK_LIBRARY=y
+CONFIG_DK_LIBRARY_INVERT_LEDS=n
+
+# Heap and stacks
+CONFIG_HEAP_MEM_POOL_SIZE=4096
+CONFIG_MAIN_STACK_SIZE=16384
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=8192
+
+CONFIG_NO_OPTIMIZATIONS=y
```

Comment:
Is it intended to build the sample w/o optimizations or is it some leftover?

Reply:
Debug leftovers, removed here and other places